_docs/_latest/setup/forseti-on-gke.md

```diff
@@ -0,0 +1,327 @@
+---
+title: Deploy Forseti Security on Google Kubernetes Engine
+order: 004
+---
+
+{% include docs/latest/alpha-release-feature.md %}
+
+# {{ page.title }}
+
+This guide explains how to setup Forseti on Kubernetes.  Most installation scenarios require the use of Terraform and the [terraform-google-forseti](https://registry.terraform.io/modules/terraform-google-modules/forseti/google/) module.  The Forseti containers are deployed on GKE using [Helm charts](https://github.com/forseti-security/helm-charts).  When using Terraform to deploy Forseti on GKE, this is transparent to the user.
+
+There are multiple scenarios for installation depending on what GCP components (GKE, Forseti) are installed, if any.  After addressing the [pre-requisites](#install-pre-requisites), please use the following table to determine from which step you should start.
```

Comment:
Please make sure this internal link works: [pre-requisites](#install-pre-requisites)